### PR TITLE
Updates for Winston 3.0

### PR DESCRIPTION
**Pull request by bcosma**
Now the log method is changed to function (info, callback).

### CODE REVIEW

**Comment by vishal-quint** [comment]
Would be really nice to see support for winston 3.x merged

File: lib/winston-papertrail.js [hunk 1]
```diff
@@ -302,9 +302,12 @@ winston.transports.Papertrail = Papertrail;
...
  * @returns {*}
  */
-Papertrail.prototype.log = function (level, msg, meta, callback) {
+Papertrail.prototype.log = function (info, callback) {
 
-    var self = this;
+    var self = this,
+        level = info.level,
+        msg = info.message,
+        meta = info.meta,;
```
Comment (realfresh) [inline lib/winston-papertrail.js]: Extra comma present after info.meta

**Review by BeauBouchard — APPROVED** [review]

**Comment by up73k** [comment]
Bump

**Comment by sqljim** [comment]
Bump as well. Any news?

**Comment by markdascher** [comment]
So far, it's unclear if this PR is the right move forward. If we're going to break compatibility with Winston 2.0, it should probably be part of a [larger rewrite](https://github.com/kenperkins/winston-papertrail/issues/84), given that winston-papertrail is [already compatible](https://github.com/kenperkins/winston-papertrail/issues/74) with Winston 3.0. Is there something in particular that this fixes?

**Comment by ffxsam** [comment]
I'm already working on a total rewrite, which includes Winston 3.x support, and support for separating the Papertrail connection from the transport(s).

**Comment by bcosma** [comment]
I am closing this pull request, since it is based on a RC version for Winston and as @markdascher said winston-papertrail is already [compatible](https://github.com/kenperkins/winston-papertrail/issues/74) with Winston 3.0

**Comment by ndelvalle** [comment]
@ffxsam did you end up creating the total rewrite? I am trying to use this library with winston 3 and is not working preperly.

**Comment by ffxsam** [comment]
Hey @ndelvalle - sorry, I'm no longer keeping tabs on this project so I don't know what state it's in.

**Pull request closed without merging**